AENS.md

```diff
@@ -125,7 +125,36 @@ This can have multiple entries, e.g. a payment address associated
 with the name and an oracle address associated with the name.
 
 
-### Name
+A subname entry (or, subdomain), belongs to a name entry.
+
+The fields **owner**, **expires_by** and **client_ttl** of the
+name entry applies to all subname entries of the name entry.
+
+The lifecycle of subnames are driven by the lifecycle of the name
+they belong to.
+
+Issuing `revoke` or `transfer` transactions on name causes revoking or
+transfering of all of the name's subnames.
+
+When the name enters `unclaimed` state, all of the name's subnames
+are deleted.
+
+Issuing `claim`, `revoke`, `transfer` and `update` transaction where
+subname is provided instead of name causes failure, and such transaction
```

Comment:
Propagation to other nodes is out of the context of this document. This is just about valid and invalid transctions.
So, better:
"such transactions do not validate".